README.md

```diff
@@ -142,6 +142,13 @@ You can use ```\#``` as an escape sequence to make ```#``` as a part of your pro
     i. in the example above ```\#``` can be used as an escape sequence to include ```#``` as a part of your prompt.
 3. #This will be considered as a comment as it contains # at the start of the line itself.
 
+##### Disabling Comments:
+
+To disable comments in prompt file (undo the behaviour discussed above), you can use -i/--ignore flag in your command for Bito CLI.   
```

Comment:
To treat # as normal character and not a special character marking starting of the comment (which is the default behavior), one can use -i/--ignore flag in your command for Bito CLI.
Using -i/--ignore flag in your command for Bito CLI will let Bito CLI know to not treat # specially and use it as part of the prompt for processing.